tgui/packages/tgui/interfaces/OverwatchConsole.jsx

```diff
@@ -87,12 +87,14 @@ const SquadPanel = (props, context) => {
             Supply Drop
           </Tabs.Tab>
         )}
-        <Tabs.Tab
-          selected={category === 'ob'}
-          icon="bomb"
-          onClick={() => setCategory('ob')}>
-          Orbital Bombardment
-        </Tabs.Tab>
+        {!!data.can_launch_obs && (
+          <Tabs.Tab
+            selected={category === 'ob'}
+            icon="bomb"
+            onClick={() => setCategory('ob')}>
+            Orbital Bombardment
+          </Tabs.Tab>
+        }}
```

Comment:
Not closing this proper - you need to close like )} not }}

Reply:
copy paste gone wrong (i made this UI originally should've caught that, oops)